scss/partials/_base.scss

```diff
@@ -184,14 +200,17 @@ dl {
   }
 
   dd {
-    @extend p,.p--body;
+    @extend p, .p--body;
```

Comment:
Why are we extending both of these? `.p--body` has no styles that make it unique from `p` at a glance.